### DISCUSSION TITLE
It is almost impossible to drag and drop

### DISCUSSION THREAD

**Discussion by ronalfy** [GitbookIO/community#431] [Feature Requests]
Is it just me or does the new interface make it almost impossible to drag and drop pages?

https://github.com/GitbookIO/community/assets/636521/f27266aa-2959-44ee-8c17-5c028378f695

**Comment by aikoii** [comment]
Hey @ronalfy! I know this is delayed 😬🙏, but are you still having issues with dragging & dropping pages?
